packages/build-utils/test/unit.test.ts

```diff
@@ -474,21 +474,21 @@ it('should support initialHeaders and initialStatus correctly', async () => {
 });
 
 it('should support require by path for legacy builders', () => {
-  const index = require('@vercel/build-utils');
+  const index = require('../dist');
```

Comment:
@TooTallNate not sure if theres a better way to do this, but these paths work and now require the project being build first since it can't rely on hoisting lol

Reply:
This test was explicitly designed to check the package boundary since we used to have builders that did this:

```
require('@vercel/build-utils/fs/get-writable-directory.js');
```

Reply:
Maybe this is slightly better. Personally I wouldn't mind deleting this test + legacy behavior entirely, but idk might be too risky.

```suggestion
  const index = require('../');
```